test/e2e/page-objects/pages/dialog/add-edit-network.ts

```diff
@@ -84,7 +84,9 @@ class AddEditNetworkModal {
    */
   async addExplorerUrl(explorerUrl: string): Promise<void> {
     console.log(`Add explorer URL ${explorerUrl}`);
-    await this.driver.clickElement(this.explorerUrlInputDropDownButton);
+    await this.driver.findScrollToAndClickElement(
+      this.explorerUrlInputDropDownButton,
+    );
```

Comment:
What is the point of this change?

Reply:
i'm removing this and put on separate PR. this is a suppose fix to a flaky e2e test

Reply:
Makes sense! Thanks for the explanation 🙏